testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeLdap.java

```diff
@@ -39,7 +39,7 @@
 public abstract class AbstractSinglenodeLdap
```

Comment:
This should be `EnvironmentExtender` that is used via composition.

Reply:
Not related to the change